node-notes-cli/updatenote.js

```diff
@@ -0,0 +1,25 @@
+const updatenote = (inputKey, inputString) => {
+
+    const fs = require('fs');
+
+    const FileName = 'data.json'
+
+    fs.readFile(FileName, 'utf8', (err, data) => {
+        if (err) {
+            console.log("Error reading File:",  err);
+        } try {
+            const content = JSON.parse(data);
+            content.notes[inputKey] = inputString;
+            const jsonString = JSON.stringify(content, null, 2);
+            fs.writeFile(FileName, jsonString, err => {
+                if (err) {
+                    console.log('Error writing JSON File: ', err);
+                }    
+            })
+        } catch (err) {
+            console.log("Error parsing JSON string: ", err);
+        } 
+    });
+}

```

Comment:
"User can update a note" shouldn't allow the user to create a new note by accident. How would you prevent that from happening?

Reply:
Fixed validation issue.
![image](https://user-images.githubusercontent.com/20628519/128444478-27f8395e-46c5-42b5-9a0c-2a999a223832.png)